panic when building image with an invalid definition file (library)

May be related to #3618 

## Version of Singularity:

master 56595fce542317ec1ebf6bfb0263bf7de26ec297

### Expected behavior

`singularity build` should not panic.

### Actual behavior

```
$ sudo singularity build busybox.sif busybox.def 
[sudo] password for gvallee: 
INFO:    Starting build...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb8 pc=0xe8b0eb]

goroutine 1 [running]:
github.com/sylabs/singularity/internal/pkg/build/sources.(*LibraryConveyorPacker).Get(0xc0001e7d00, 0xc000496240, 0x0, 0x1)
        internal/pkg/build/sources/conveyorPacker_library.go:67 +0x51b
github.com/sylabs/singularity/internal/pkg/build.(*Build).Full(0xc0001ec280, 0x0, 0x0)
        internal/pkg/build/build.go:198 +0x4b4
github.com/sylabs/singularity/cmd/internal/cli.run(0x18aa840, 0xc00046f980, 0x2, 0x2)
        cmd/internal/cli/build_linux.go:146 +0x474
github.com/spf13/cobra.(*Command).execute(0x18aa840, 0xc0000300e0, 0x2, 0x2, 0x18aa840, 0xc0000300e0)
        vendor/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0x18b3200, 0x8116cb, 0x7f13e4d13000, 0x0)
        vendor/github.com/spf13/cobra/command.go:850 +0x2f8
github.com/sylabs/singularity/cmd/internal/cli.ExecuteSingularity()
        cmd/internal/cli/singularity.go:230 +0x133
main.main()
        cmd/singularity/cli.go:17 +0x20
```
### Steps to reproduce behavior


```
$ cat busybox.def 
Bootstrap: library
From: ubuntu/latest
```

--- a/internal/pkg/build/sources/conveyorPacker_library.go
+++ b/internal/pkg/build/sources/conveyorPacker_library.go
@@ -58,9 +58,15 @@ func (cp *LibraryConveyorPacker) Get(b *types.Bundle) (err error) {
 
 	imageRef := library.NormalizeLibraryRef(b.Recipe.Header["from"])
 
-	libraryImage, _, err := libraryClient.GetImage(context.TODO(), imageRef)
+	libraryImage, existOk, err := libraryClient.GetImage(context.TODO(), imageRef)
 	if err != nil {
-		return err
+		return fmt.Errorf("while getting image info: %v", err)
+	}
+	if !existOk {
+		return fmt.Errorf("image does not exist in the library")
+	}
+	if libraryImage == nil {
+		return fmt.Errorf("failed getting image from the library")
 	}
 
 	imageName := uri.GetName("library://" + imageRef)

